tests/beaker_tests/cisco_acl/test_ace.rb

```diff
@@ -0,0 +1,222 @@
+#
+# The test cases use RegExp pattern matching on stdout or output IO
+# instance attributes to verify resource properties.
+#
+###############################################################################
+
+# Require UtilityLib.rb paths.
+require File.expand_path('../../lib/utilitylib.rb', __FILE__)
+
+# -----------------------------
+# Common settings and variables
+# -----------------------------
+testheader = 'Resource cisco_ace'
+
+# 'tests' hash
+# Top-level keys set by caller:
+# tests[:master] - the master object
+# tests[:agent] - the agent object
+# tests[:show_cmd] - the common show command to use for test_show_run
+#
+tests = {
+  master:   master,
+  agent:    agent,
+  show_cmd: 'show run aclmgr',
+}
+
+# tests[id] keys set by caller and used by test_harness_common:
+#
+# tests[id] keys set by caller:
+# tests[id][:desc] - a string to use with logs & debugs
+# tests[id][:manifest] - the complete manifest, as used by test_harness_common
+# tests[id][:resource] - a hash of expected states, used by test_resource
+# tests[id][:resource_cmd] - 'puppet resource' command to use with test_resource
+# tests[id][:show_pattern] - array of regexp patterns to use with test_show_cmd
+# tests[id][:ensure] - (Optional) set to :present or :absent before calling
+# tests[id][:code] - (Optional) override the default exit code in some tests.
+#
+# These keys are local use only and not used by test_harness_common:
+#
+# tests[id][:manifest_props] - This is essentially a master list of properties
+#   that permits re-use of the properties for both :present and :absent testing
+#   without destroying the list
+# tests[id][:resource_props] - This is essentially a master hash of properties
+#   that permits re-use of the properties for both :present and :absent testing
+#   without destroying the hash
+# tests[id][:title_pattern] - (Optional) defines the manifest title.
+#   Can be used with :af for mixed title/af testing. If mixing, :af values will
+#   be merged with title values and override any duplicates. If omitted,
+#   :title_pattern will be set to 'id'.
+#
+
+tests['default_properties_ipv4'] = {
+  title_pattern:  'ipv4 foo-1',
+  manifest_props: "
+    stats_per_entry                  => 'false',
+    fragments                        => 'default',
+  ",
+  resource_props: {
+    'stats_per_entry' => 'false',
+    'fragments'       => '',
+  },
+}
+
+tests['create_ipv4_seq_10'] = {
+  title_pattern:  'ipv4 beaker_1 10',
+  manifest_props: "
+    'action'        => 'permit',
+    'proto'         => 'tcp',
+    'src_addr'      => '1.2.3.4 2.3.4.5',
+    'src_port'      => 'eq 40',
+    'dst_addr'      => '9.9.0.4/32',
+    'dst_port'      => 'range 32 56',
+  ",
+  resource_props: "
+    'action'        => 'permit',
+    'proto'         => 'tcp',
+    'src_addr'      => '1.2.3.4 2.3.4.5',
+    'src_port'      => 'eq 40',
+    'dst_addr'      => '9.9.0.4/32',
+    'dst_port'      => 'range 32 56',
+  ",
+}
+
+tests['destroy_ipv4_seq_10'] = {
```

Comment:
My comment from #121 still needs to be addressed.

https://github.com/cisco/cisco-network-puppet-module/pull/121/files#diff-ac20693befc9cf1b0135cf1ba18823a0R131